web/profiles/custom/os2loop/modules/os2loop_upvote/os2loop_upvote.module

```diff
@@ -0,0 +1,30 @@
+<?php
+
+/**
+ * @file
+ * The module file for os2loop_upvote.
+ */
+
+/**
+ * Use service to add number of upvotes to flag.
+ */
+function os2loop_upvote_preprocess_flag(array &$variables) {
```

Comment:
We should just delegate to a generic `preprocessFlag` function in the helper to keep any logic out of the actual hook implementation.